### PR TITLE
➕ Add definition for spiritual bypassing

### LINKED ISSUE #269: Suggestion: spiritual bypassing
As per John Welwood, by way of [Psychology Today](https://www.psychologytoday.com/us/blog/the-empowerment-diary/201901/what-is-spiritual-bypassing): “using ‘spiritual ideas and practices’ to sidestep personal, emotional ‘unfinished business,’ to shore up a shaky sense of self, or to belittle basic needs, feelings, and developmental tasks.”

In addition to concerns around emotional labor, there is an angle of co-opting the language, beliefs, and practices of India. There are also considerations for downplaying BIPOC concerns and criticism via pushing for "good vibes only."

I could take a shot at writing an initial term, but I feel it would need review from someone with more familiarity with Indian culture and Hindu faith.

### PR DESCRIPTION
**Pull request by ericwbailey**
This PR addresses https://github.com/selfdefined/web-app/issues/269. It adds a definition for the term spiritual bypassing.

As noted in the issue, I would like a review from someone with more familiarity with Indian culture and Hindu faith to make sure my terminology and phrasing are accurate.

### CODE REVIEW

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
title: spiritual bypassing
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
excerpt: a form of appropriation where non-Black individuals use memes and imagery featuring Black people and dark-skinned emojis
```
I think this needs updating

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
a behavior that uses spiritual ideas and practices to delay or ignore acknowledging hardship; the pursuit of “good vibes only” or “oneness” that can downplay difficulties brought up by others, especially from [minoritized](/definitions/minoritised/) groups; present in many Western wellness and self-help practices, primarily Yoga
```

Comment (tatianamac) [reply 11ty/definitions/spiritual-bypassing.md]: I provided a heavy-handed edit here with the definition because it restates parts of the word; generally we treat this like a "dictionary" definition with broken-out phrases.

One area that needs rewrite is in the last clause starting with "present in..." The way it's written implies that Yoga is a Western wellness practice. If we're going to say something to that effect, and I understand where you're going with it, we need to address that it's the Western and white culture appropriation of Yoga. Without that delineation, it reads that we think Yoga is a white and Western practice.

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Not addressing discomfort means an individual will not process and internalize difficult or painful events. Escapism of this form prevents the individual from working through discomfort and hardship. Such a spiritual bypassing approach is directly at odds with Yoga’s practice of being present and working through emotional discomfort.
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Furthermore, co-opted language and imagery is often drawn specifically from Indian culture and Hindu faith. Concepts are often presented by people who do not identify as Indian or of Hindu faith and without attribution to and context of or in conflict with the original cultural context.
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md:45]: Telling someone to "feel better" or to "get well" or to suggest Yoga or other unsolicited medical advice when not someone's explicit medical adviser. Such comments can be hurtful and damaging to mentally and physically chronically ill and disabled people who cannot and will not "get better" or "get well."

**Review by tatianamac — CHANGES_REQUESTED** [review]

File: 11ty/definitions/spiritual-bypassing.md
Comment (ericwbailey) [reply 11ty/definitions/spiritual-bypassing.md]: Oh whoops, sorry about that. Guess which term I duplicated to get started? 😬

File: 11ty/definitions/spiritual-bypassing.md
Comment (ericwbailey) [reply 11ty/definitions/spiritual-bypassing.md:45]: I like this example. Do you mind if I incorporate it?

File: 11ty/definitions/spiritual-bypassing.md
Comment (ericwbailey) [reply 11ty/definitions/spiritual-bypassing.md]: Thank you for this edit and its feedback, I think it reads way better now. I also agree with your comment about the last clause. I'll work to update it.

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [reply 11ty/definitions/spiritual-bypassing.md:45]: I’d love that!

**Comment by jeevajay** [comment]
Hi @ericwbailey,

I was able to spend some time on this issue. I've already commented on #269 agreeing that Indian Culture and Hinduism predominantly influences co-opting language and the prevalence of spiritual bypassing in our society. Below I'm trying to raise a few concerns and add some suggestions without my personal bias.

**Concerns**

Concern 1: Co-opting language can be used by any Spiritual Teachers, Spiritual Leaders, Preachers, Self-help Gurus, etc. But the issues focus on Wellness Instructors and Yoga Gurus. It might seem that the view is biased against them.

Concern 2: Because issues focus specifically on Yoga Instructors, the influence of Indian Culture and Hinduism is mentioned. In addition to bias against Yoga Instructors, it might also look Indian Culture and Hinduism is singled out while spiritual bypassing is prevalent in other cultures and religions too.

Concern 3: Only Wellness Spaces are mentioned as space where spiritual bypassing can be perpetrated against the minoritized groups. What about other places: Temple, Church, Hospital, Therapy Groups, Self-help Groups?

I want to be clear that by raising these concerns I'm not saying that these spaces or religions shouldn't be called out. I am asking why all other spiritual and religious places aren't called out too?

--

**Suggestion:** Whether showing the ill effects of Spiritual Bypassing would be helpful for the readers? Adding a para that could be useful from the research paper I read: [https://www.researchgate.net/publication/319321341_The_Opiate_of_the_Masses_Measuring_Spiritual_Bypass_and_Its_Relationship_to_Spirituality_Religion_Mindfulness_Psychological_Distress_and_Personality](https://www.researchgate.net/publication/319321341_The_Opiate_of_the_Masses_Measuring_Spiritual_Bypass_and_Its_Relationship_to_Spirituality_Religion_Mindfulness_Psychological_Distress_and_Personality)

Page 2 Col 2:

> Possible negative consequences of spiritual bypass include the need to control others and self, dichotomous thinking, shame, spiritual obsession, fear, emotional confusion, addiction, high tolerance for inappropriate behavior, codependence, pain compulsive goodness, narcissism or ego inﬂation, obsession or addiction, blind belief in charismatic teachers, spiritual materialism (use of spiritual practice for material gain), developmental arrest, and abdication of personal responsibility.

**Question:** In the What to Do Instead section, it is suggested people should call out when they recognize "spiritual bypassing as a discreet behavior". Did you mean to write that people should call out co-opting language? Because calling out co-opting language is different from calling out spiritual bypassing. People who are spiritual bypassing are already suppressing the need to address their psychological needs, consciously, or unconsciously. Strangers calling them out on a public forum might do more harm than good.

Related text from the above-referenced paper. Page 12 Col 1:

> Given that spiritual bypass is typically implicit and unconscious, for many clients this process may begin simply with seeing their spiritual bypass in more explicit and conscious ways. Because the client has used strategies, albeit unconsciously, for avoidance, it is predictable that most clients will experience some resistance and ambivalence in this process, which should be addressed with great respect using the motivational interviewing framework referenced above, honoring that the spiritual bypass has helped the client to cope yet also is maladaptive in other ways. Power struggles with clients will almost certainly be ineffective.

This is a very important topic to cover. Thanks for creating this. Looking forward to your responses.

**Comment by ericwbailey** [comment]
@jeevajay This is incredible feedback, thank you. Apologies for the delay in replying—I think everything you've outlined are great insights. I'll work to update the term, and mention you here when I've pushed up changes.

**Comment by jeevajay** [comment]
Hi @ericwbailey, happy to be involved. I wanted to add that "Co-opting" can also be defined separately so it can be connected to many other definitions where co-opting occurs, apart from its influence on this topic. Do share your thoughts when you can. Thanks.

**Comment by tatianamac** [comment]
@ericwbailey @jeevajay Hi! Do you all have the capacity/desire to update this term? Totally understand if not, especially given everything going on in the world. No pressure on my side—I'm mostly hoping to just update the status of the terms in queue. thank you! 🙏🏽

**Comment by jeevajay** [comment]
Hi @tatianamac, as this term was suggested by @ericwbailey, I will wait for his confirmation. Otherwise, I would like to work on it. Personally, if you remember I want to update the OCD definition first, I've spent time and done the work to understand different views, issues, and impacts. I just couldn't sit and complete it in the last month. I take this message as an encouragement to push myself to complete that. In the meantime, based on Eric's reply I will work on this. I'm also thinking of asking for an Accountability Partner or Thought Buddy on the Slack channel to collaborate with anyone who might be interested. Is that okay?

**Comment by tatianamac** [comment]
@jeevajay Thank you for your response! Totally makes sense to defer to @ericwbailey ! Re: the OCD term, yes, I'd love your thoughts/improvements there if and when you have bandwidth! And by all means, please feel free to ask the #general channel—it's a very lovely idea and I imagine someone would find that helpful too! Take care!

**Comment by ericwbailey** [comment]
Hi @tatianamac. I definitely have interest in carrying this through. Covid, 2020, and client work gave me some setbacks, but that's on me for not communicating. I'll get @jeevajay's feedback incorporated soon.

And speaking of, @jeevajay I'd be more than happy to be a accountability buddy on Slack if you want 🙂

**Comment by jeevajay** [comment]
Thanks for the encouragement @tatianamac -- will share the thought on Slack. @ericwbailey will msg you on Slack. Thanks much.

**Comment by tatianamac** [comment]
Thanks for the update, @ericwbailey ! No worries—totally understand times are rough.

**Comment by ericwbailey** [comment]
@jeevajay I've just pushed up copy changes based on your feedback. There's some sizable restructuring based on our conversation earlier today, and I think the definition is in a far better shape because of it. This is to say thank you so much for your help! I really enjoyed our cowriting session 🙂

I'm also more than happy to make further edits, if desired.

**Comment by jeevajay** [comment]
Hi @ericwbailey this looks great. I think we can merge and close this. @tatianamac pls review. Thanks.

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
a behaviour that uses spiritual ideas and practices to delay or ignore acknowledging hardship; the pursuit of “good vibes only” or “oneness” that can downplay difficulties brought up by others, especially from [minoritised](/definitions/minoritised/) groups; commonly exhibited by Western wellness and self-help practitioners, including spiritual teachers and leaders, Yoga instructors, preachers, and motivational speakers.
```
Changing grammar here to align with "behaviour"

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Saying a phrase such as, “we are all one” or “we are all human” when confronted with difficulty or hardship, in particular when that hardship or oppression is specific to an axis of someone's identity, such as their race, gender, neurodiversity, ability, etc.
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Not addressing discomfort means an individual will not process and internalise difficult or painful events. Escapism of this form prevents the individual from working through hardship. 
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Failing to acknowledge hardship is especially problematic in the context of minoritised groups. Downplaying invalidates both their lived experiences and creates an environment that both disincentives further participation and suppresses trauma.
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [reply 11ty/definitions/spiritual-bypassing.md:45]: I think in this section we need to address power and privilege. Often the people telling someone to employ "good vibes" has systemic power and privilege. So, them telling someone who experiences institutional oppression to just "have good vibes" is an abuse of their power as well, invalidating their lived experiences.

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Identification means an individual can cease these behaviors, if practicing them. If witnessed, it is advised to use indirect questioning techniques such as [motivational interviewing](https://en.wikipedia.org/wiki/Motivational_interviewing) to ascertain the underlying factors compelling this behaviour.

Direct confrontation may have opposite of the desired effect. It may be construed as an attack on the person's authority, or cause them to double-down and further incorporate this behaviour into their sense of identity and cause even more harm.
```

**Review by tatianamac — APPROVED** [review]
Thank you, @ericwbailey and @jeevajay! The changes are really great. 

One last editorial note I have is with regards to power. I wonder if we could do more to tease out that often the people conducting this sort of behaviour have systemic privilege and power over the people they're seeking to _regulate_—it's an abuse of power and invalidates vulnerable people's experiences. I feel we would be remiss to omit this power dynamic.

Beyond that I changed a few grammar notes and converted into International English. 👍🏽 

We're super close!

File: 11ty/definitions/spiritual-bypassing.md
Comment (ericwbailey) [reply 11ty/definitions/spiritual-bypassing.md]: u werd gud

**Comment by ericwbailey** [comment]
@tatianamac I just pushed up an attempt to address your feedback about regulating behavior. The language comes at the end of the Issues section, and not with https://github.com/selfdefined/web-app/pull/292#discussion_r529934072. Happy to move it if you feel it's more appropriate.

@jeevajay I'd also welcome a review of the new copy if you have the time.

**Comment by jeevajay** [comment]
Hi @ericwbailey last week was disrupted due to a severe cyclone here in southern India. No power and internet for a few days, so I had to catch up with work over the weekend. I will look into it and confirm by tomorrow. Thanks.

**Comment by ericwbailey** [comment]
@jeevajay No rush. Take care of yourself and your loved ones first.

**Comment by jeevajay** [comment]
Hi @ericwbailey how are you? Definition with the edits looks good. We can move ahead, I believe.

**Comment by ericwbailey** [comment]
I'm doing okay, @jeevajay. Hope you and yours are too. And thank you for the approval.

Back to you for review, @tatianamac.

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
excerpt: a behaviour that uses spiritual ideas and practices to delay or ignore acknowledging hardship especially of minoritised groups
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Also understand the chilling effects it has on minoritised communities, and how it affects their presence (or lack of presence) in spaces where spiritual bypassing is practiced.
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [reply 11ty/definitions/spiritual-bypassing.md:45]: @ericwbailey Did you all opt no to include this example? Fine if it was intentional but want to flag in case it was not. (I find it adds an important dimension of disableism myself?)

**Review by tatianamac — APPROVED** [review]
🚀 A few grammatical notes (conversion to Int'l English), removing trailing spaces (which will resolve our build error), and one content question! Very close. Thank you so much!

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Spiritual bypassing avoids deeply addressing discomfort by offering surface-level platitudes, often in the form of co-opted religious, philosophical, and spiritual practices, terminology, and imagery.
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Not addressing discomfort means an individual will not process and internalise difficult or painful events. Escapism of this form prevents the individual from working through hardship.
```

File: 11ty/definitions/spiritual-bypassing.md
Comment (tatianamac) [inline 11ty/definitions/spiritual-bypassing.md]:
```suggestion
Identification means an individual can cease these behaviors, if practising them. If witnessed, it is advised to use indirect questioning techniques such as [motivational interviewing](https://en.wikipedia.org/wiki/Motivational_interviewing) to ascertain the underlying factors compelling this behaviour.
```

**Comment by tatianamac** [comment]
@jeevajay I hope things are okay with you and your loved ones! In my thoughts. Thank you for all you've done!

@ericwbailey Thanks for ushering this definition along. 

Very excited to launch this definition thanks to both of your collaboration!

**Comment by jeevajay** [comment]
@tatianamac All good. Thank you!
@ericwbailey congrats!

Looking forward to the merge.

**Comment by ericwbailey** [comment]
@tatianamac Accepted your suggestions, thanks for creating them! I think I have https://github.com/selfdefined/web-app/pull/292#discussion_r529934072 addressed via [the preceding paragraph](https://github.com/selfdefined/web-app/pull/292/files#diff-52fd56da473ab4607f647880310b1aeb674cffe692dccee256f6fb29d8a8339bR30), happy to further elaborate if you feel it needs, it however.

**Review by tatianamac — APPROVED** [review]
🚀 Approved!

**Pull request merged**
